src/components/Banner/tests/Banner.test.tsx

```diff
@@ -0,0 +1,197 @@
+            const spy = jest.fn();
+            const onDismiss: Action = {content: 'Action Content', onAction: () => { spy(); } };
+            const bannerWrapper = mount(<Banner title="Order archived" action={onDismiss}>
+                                        <p>Use your finance report.
+                                        <Link url="http://google.com">
+                                        Let us know what you think.</Link></p>
+                                        </Banner>);
+            bannerWrapper.find('button').simulate('click');
+            expect(spy).toBeCalled();
+        });
+    });
+
+    describe('Verify Banner component different status', () => {
+        it('should verify banner status set as default', () => {
+            const bannerWrapper = mount(<Banner title="Order archived" >
+                                        <p>This order was archived on March 7.</p>
+                                        </Banner>);
+            expect(bannerWrapper.prop('title')).toBe('Order archived');
+        });
+
+        it('should verify banner status set as success', () => {
+            const bannerWrapper = mount(<Banner title="Order archived" status="success" >
+                                        <p>This order was archived on March 7.</p>
+                                        </Banner>);
+            expect(bannerWrapper.prop('status')).toBe('success');
+        });
+
+        it('should verify banner status set as info', () => {
+            const bannerWrapper = mount(<Banner title="Order archived" status="info" >
+                                        <p>This order was archived on March 7.</p>
+                                        </Banner>);
+            expect(bannerWrapper.prop('status')).toBe('info');
+        });
+
+        it('should verify banner status set as warning', () => {
+            const bannerWrapper = mount(<Banner title="Order archived" status="warning" >
+                                        <p>This order was archived on March 7.</p>
+                                        </Banner>);
+            expect(bannerWrapper.prop('status')).toBe('warning');
+        });
+
+        it('should verify banner status set as critical', () => {
+            const bannerWrapper = mount(<Banner title="Order archived" status="critical" >
+                                        <p>This order was archived on March 7.</p>
+                                        </Banner>);
+            expect(bannerWrapper.prop('status')).toBe('critical');
+        });
+
+        it('should verify banner status set as onDismiss', () => {
+            const bannerWrapper = mount(<Banner onDismiss={ () => {} } >
+                                        <p>Use your finance report. 
+                                        <Link url="">Let us know what you think.</Link></p>
+                                        </Banner>);
+            expect(typeof bannerWrapper.prop('onDismiss')).toBe('function');
```

Comment:
This is testing for type of onDismiss as function but it does not test for "should verify banner status set as onDismiss"